lib/rubocop/cop/style/method_call_with_args_parentheses.rb

```diff
@@ -153,6 +157,7 @@ def add_offense_for_require_parentheses(node)
 
         def add_offense_for_omit_parentheses(node)
           return unless node.parenthesized?
+          return if super_call_without_arguments?(node)
```

Comment:
I think the boundaries between `Style/MethoCallWithArgsParentheses` and `Style/MethoCallWithoutArgsParentheses`. I'm linting part of the method calls _without_parentheses here. Maybe I can move this check to the `Style/MethoCallWithoutArgsParentheses` and leave the no-args checks for it?

Or! Maybe we can combine the two cops eventually?

Reply:
Yeah, maybe. Before the new style there was no overlap, so I never really thought much about this. I'm ok with the current arrangement, though.